openapi/index.yaml

```diff
@@ -684,6 +718,17 @@ parameters:
     description: A date in the format YYYY-MM-DD.
     pattern: "[0-9]{4}-[0-9]{2}-[0-9]{2}"
     x-example: "2019-09-15"
+  OrganizationFiscalCode:
+    name: organization_fiscal_code
+    in: path
+    type: string
+    description: Organization fiscal code.
+    format: OrganizationFiscalCode
+    minLength: 11
+    maxLength: 11
+    pattern: "^[0-9]{11}$"
+    required: true
+    x-example: "12345678901"
```

Comment:
you should use this format:

https://github.com/pagopa/io-functions-commons/blob/master/openapi/definitions.yaml#L290